contrib/mesos/pkg/executor/timer.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package executor
+
+import (
+	"time"
+)
+
+// taskTimer is a convenience wrapper around time.Timer
+type taskTimer time.Timer
```

Comment:
the semantics are different. it keep the code cleaner elsewhere. i'll write up some docs. it's a construct with a purpose and, given it's very small size, i'd argue quite strongly against being "over-engineered".